runtime-k3d/src/deployment/deployment.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+
+from vehicleapp import build_vehicleapp, deploy_vehicleapp
+from yaspin import yaspin
+
+
+def deployment(action: str):
+    """Execute deployment action for VehicleApp."""
+    with yaspin(text=f"{action.capitalize()}ing VehicleApp...") as spinner:
+        try:
+            if action == "build":
+                build_vehicleapp(spinner)
+                spinner.ok()
+            elif action == "deploy":
+                deploy_vehicleapp(spinner)
+                spinner.ok()
+        except Exception as err:
+            spinner.fail(err)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Specify deployment action for VehicleApp"
+    )
+    parser.add_argument(
```

Comment:
Instead of this parameter, why don't we just have `build_vehicleapp.py` and `deploy_vehicleapp.py`?